app/styles/_accounts.scss

```diff
@@ -196,3 +196,17 @@
     width: 62px;
   }
 }
+
+.account-footer {
+  @include flexbox((
+    display: flex,
+    justify-content: space-around
```

Comment:
I've had some cross-browser issues with `justify-content: space-around` on when I was working on the persistent player. Looks okay here on FF, Chrome, and Safari, but if we get any tickets later regarding footer weirdness in mobile browsers or edge this might be a good place to start.